controlplane/test/webhooks.test.ts

```diff
@@ -0,0 +1,273 @@
+import { EnumStatusCode } from '@wundergraph/cosmo-connect/dist/common/common_pb';
+import { afterAll, beforeAll, describe, expect, test } from 'vitest';
+import { OrganizationEventName } from '@wundergraph/cosmo-connect/dist/notifications/events_pb';
+import { afterAllSetup, beforeAllSetup, genID, TestUser } from '../src/core/test-util.js';
+import { SetupTest } from './test-util.js';
+
+let dbname = '';
+
+describe('Webhooks', (ctx) => {
+  beforeAll(async () => {
+    dbname = await beforeAllSetup();
+  });
+
+  afterAll(async () => {
+    await afterAllSetup(dbname);
+  });
+
+  test('Should be able to create a webhook for a federated graph', async (testContext) => {
+    const { client, server } = await SetupTest({ dbname });
+
+    const fedGraphName = genID('fedGraph');
+
+    const createFedGraphRes = await client.createFederatedGraph({
+      name: fedGraphName,
+      namespace: 'default',
+      routingUrl: 'http://localhost:8081',
+    });
+
+    expect(createFedGraphRes.response?.code).toBe(EnumStatusCode.OK);
+
+    const graph = await client.getFederatedGraphByName({
+      name: fedGraphName,
+      namespace: 'default',
+    });
+
+    if (!graph.graph) {
+      throw new Error('Graph could not be found');
+    }
+
+    expect(graph.response?.code).toBe(EnumStatusCode.OK);
+    expect(graph.graph?.name).toBe(fedGraphName);
+
+    await client.createOrganizationWebhookConfig({
+      endpoint: 'http://localhost:8081',
+      eventsMeta: [
+        {
+          eventName: OrganizationEventName.FEDERATED_GRAPH_SCHEMA_UPDATED,
+          meta: {
+            case: 'federatedGraphSchemaUpdated',
+            value: {
+              graphIds: [graph?.graph?.id],
+            },
+          },
+        },
+      ],
+    });
+
+    await server.close();
+  });
+
+  test('Should be possible to subscribe for a federated graph that dont belong to the same organization', async (testContext) => {
```

Comment:
Should it be "Should be possible to subscribe for a federated graph that belong to the same organization"

Reply:
Indeed, Good catch!